scripts/flash.bat

```diff
@@ -0,0 +1,14 @@
+set mac=$(python -m esptool.py read_mac | grep -m 1 'MAC:' | grep -o '.\{8\}$' )
```

Comment:
Same.

Reply:
Yeah sorry this file was pushed in by accident